### DISCUSSION TITLE
maca data download running indefinitely

### DISCUSSION THREAD

**Discussion by allenmoench** [earthlab-education/Earth-Analytics-AY24#652] [Q&A]
Hello! I'm having an issue downloading the maca data. I used Elsa's video to set up the code, but when I run it, it gets hung up indefinitely and never completes. I used the same data link that Elsa used in the video. I'm also sometimes getting a "FileNotFoundError: [Errno 2] No such file or directory", which makes me think maybe it's an issue with the data source / url not leading anywhere? In the image below, I had to manually terminate the code.

![image](https://github.com/user-attachments/assets/e82c9bb0-1105-4310-9e58-0aa7250fcb5c)

**Comment by nquarder** [comment]
@allenmoench see if you can track down @eculler at AGU this week. 

Either at her poster today (Weds) from 13:40-17:30

https://agu.confex.com/agu/agu24/meetingapp.cgi/Paper/1741876

Or tomorrow (Thurs) from 08:30-12:20

https://agu.confex.com/agu/agu24/meetingapp.cgi/Paper/1743498

I bet she can help get your question answered! Have fun at AGU this week and once you get an answer remember to post it here 😸 🌍

**Comment by akpost21** [comment]
Hmm @allenmoench that loop looks correct to me (but I might be missing something!). Have you tried doing each step separately outside of a loop? Can you access the url on it's own?
